Swiftfin tvOS/Views/SettingsView/VideoPlayerSettingsView.swift

```diff
@@ -43,46 +43,54 @@ struct VideoPlayerSettingsView: View {
             .contentView {
 
                 Section {
+
                     ChevronButton(
-                        "Resume Offset",
+                        L10n.offset,
                         subtitle: resumeOffset.secondLabel
                     )
                     .onSelect {
                         isPresentingResumeOffsetStepper = true
                     }
+                } header: {
+                    L10n.resume.text
                 } footer: {
-                    Text("Resume content seconds before the recorded resume time")
+                    L10n.resumeOffsetDescription.text
                 }
 
                 Section {
+
                     ChevronButton(L10n.subtitleFont, subtitle: subtitleFontName)
                         .onSelect {
                             router.route(to: \.fontPicker, $subtitleFontName)
                         }
+                } header: {
+                    L10n.subtitles.text
                 } footer: {
-                    Text("Settings only affect some subtitle types")
+                    L10n.subtitlesDisclaimer.text
                 }
 
                 Section {
-
-                    Toggle("Pause on background", isOn: $pauseOnBackground)
-                    Toggle("Play on active", isOn: $playOnActive)
-                }
-            }
-            .navigationTitle("Video Player")
-            .blurFullScreenCover(isPresented: $isPresentingResumeOffsetStepper) {
-                StepperView(
-                    title: "Resume Offset",
-                    description: "Resume content seconds before the recorded resume time",
-                    value: $resumeOffset,
-                    range: 0 ... 30,
-                    step: 1
-                )
-                .valueFormatter {
-                    $0.secondLabel
+                    Toggle(L10n.pauseOnBackground, isOn: $pauseOnBackground)
+                    Toggle(L10n.playOnActive, isOn: $playOnActive)
+                } header: {
```

Comment:
Please use other init.

Reply:
I believe this is the same item as https://github.com/jellyfin/Swiftfin/pull/1163#discussion_r1706031863 so I will base this change on that discussion!

Reply:
https://github.com/jellyfin/Swiftfin/pull/1163#discussion_r1706053161